.github/workflows/build.yaml

```diff
@@ -103,10 +104,33 @@ jobs:
           file: ./Dockerfile  # Path to your Dockerfile
           push: false
           tags: ${{ steps.meta.outputs.tags }}
+          load: true
           outputs: type=oci,compression=gzip,dest=${{ steps.vars.outputs.image_path }}-${{ steps.vars.outputs.tag_hash }}.tar.gz
           cache-from: type=local,src=${{ steps.vars.outputs.cache_dir }}
           cache-to: type=local,mode=max,image-manifest=true,oci-mediatypes=true,dest=${{ steps.vars.outputs.cache_dir }}
 
+      # # Keeping this here since it's really useful when debugging this gha
```

Comment:
Maybe make this comment more clear, like: "Uncomment to ... when debugging integration tests", etc.